_episodes/03-monitor-the-model.md

```diff
@@ -431,6 +460,18 @@ NN RMSE: 4.05, baseline RMSE: 3.88
 
 Judging from the numbers alone, our neural network preduction would be performing worse than the baseline.
 
+> ## Exercise: Baseline
+> Looking at this baseline: Would you consider this a simple or a hard problem to solve?
+>
+> > ## Solution
+> >  
+> > This really depends on your definition of hard! The baseline gives a more accurate prediction than just
+> > randomly predicting a number, so the problem is not impossible. However, if no model is going to
+> > perform better than the baseline, the problem may still be very hard.
```

Comment:
The last sentence of this solution appears quite confusing to me. Just a suggestion:

```suggestion
> > This really depends on your definition of hard! The baseline gives a more accurate prediction than just
> > randomly predicting a number, so the problem is not impossible to solve with machine learning. However, given the structure of the data and our expectations with respect to quality of prediction, it may remain hard to find a good algorithm which exceeds our baseline by orders of magnitude. 
```